corehq/form_processor/tests/test_run_sql.py

```diff
@@ -1,279 +0,0 @@
-from __future__ import unicode_literals
-from __future__ import absolute_import
-from datetime import datetime, timedelta
-from io import BytesIO
-from pprint import pformat, pprint
-from uuid import uuid4
-
-from django.db import connections
-from django.test import TransactionTestCase
-
-from corehq.blobs import CODES
-from corehq.blobs.models import BlobMeta
-from corehq.blobs.tests.util import new_meta, TemporaryFilesystemBlobDB
-from corehq.form_processor.management.commands.run_sql import (
-    Command,
-    get_form_attachment_blob_metas_by_key,
-)
-from corehq.form_processor.models import (
-    DeprecatedXFormAttachmentSQL,
-    XFormInstanceSQL,
-)
-from corehq.sql_db.util import (
-    get_db_alias_for_partitioned_doc,
-    get_db_aliases_for_partitioned_query,
-    new_id_in_same_dbalias,
-)
-
-
-class TestRunSql(TransactionTestCase):
-    # this can be removed with form_processor_xformattachmentsql table
```

Comment:
Does this mean we are in a place where we can remove this model? I think that's true, but just wanted to verify

Reply:
I think yes. It's probably been long enough and we are sufficiently confident that we will not need the data in that table for anything else (the migration appears to have been successful).